From 771016f0705909c908917b4ccaeafc950ba93c05 Mon Sep 17 00:00:00 2001
From: Dave Airlie <airlied@gmail.com>
Date: Mon, 16 Nov 2015 09:05:26 +1000
Subject: modesetting: drop platform_dev pointer.

This isn't used anywhere, so no point storing it until we need it.

Reviewed-by: Mark Kettenis <kettenis@openbsd.org>
Signed-off-by: Dave Airlie <airlied@redhat.com>

diff --git a/hw/xfree86/drivers/modesetting/driver.c b/hw/xfree86/drivers/modesetting/driver.c
index 80abcdf..00f966c 100644
--- a/hw/xfree86/drivers/modesetting/driver.c
+++ b/hw/xfree86/drivers/modesetting/driver.c
@@ -388,7 +388,6 @@ ms_platform_probe(DriverPtr driver,
         {
             DevUnion *pPriv;
             EntityInfoPtr pEnt;
-            modesettingEntPtr pMSEnt;
 
             xf86SetEntitySharable(entity_num);
 
@@ -401,13 +400,8 @@ ms_platform_probe(DriverPtr driver,
 
             xf86SetEntityInstanceForScreen(scrn, pEnt->index, xf86GetNumEntityInstances(pEnt->index) - 1);
 
-            if (!pPriv->ptr) {
+            if (!pPriv->ptr)
                 pPriv->ptr = xnfcalloc(sizeof(modesettingEntRec), 1);
-                pMSEnt = pPriv->ptr;
-            } else {
-                pMSEnt = pPriv->ptr;
-            }
-            pMSEnt->platform_dev = dev;
         }
 
     }
diff --git a/hw/xfree86/drivers/modesetting/driver.h b/hw/xfree86/drivers/modesetting/driver.h
index 14137c1..5e1c5d9 100644
--- a/hw/xfree86/drivers/modesetting/driver.h
+++ b/hw/xfree86/drivers/modesetting/driver.h
@@ -61,9 +61,6 @@ typedef struct
     unsigned long fd_wakeup_registered; /* server generation for which fd has been registered for wakeup handling */
     int fd_wakeup_ref;
     unsigned int assigned_crtcs;
-#ifdef XSERVER_PLATFORM_BUS
-    struct xf86_platform_device *platform_dev;
-#endif
 } modesettingEntRec, *modesettingEntPtr;
 
 typedef void (*ms_drm_handler_proc)(uint64_t frame,
-- 
cgit v0.10.2

